.github/workflows/tests.yaml

```diff
@@ -19,14 +19,15 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install --no-install-recommends -y graphviz make wget
-          python -m pip install black==20.8b1 mdformat==0.7.5
+          python -m pip install black flake8 isort mdformat
 
-      - name: Check fmt
+      - name: Check formatting
         run: |
-          mdformat --check .
           black --check .
+          flake8 --max-doc-length=88 --max-line-length=88 .
+          isort --check-only --multi-line=3 --profile=black .
+          mdformat --check --wrap=88 .
```

Comment:
I tried to add all checks/linting we perform.

Reply:
I think we shouldn't run a `flake8` and `isort` on snippets. I think this meant for code bases. I can imagine that I want to show bad code on purpose as a counter-example.

Reply:
Saw your comment below, I will fix that.

Reply:
Removed `flake8` and `isort`.